README.md

```diff
@@ -1,10 +1,12 @@
 HTTP Factories
 ==============
 
-This repository holds all interfaces related to [PSR-17 (HTTP Message Factories)][psr-17]. 
-Please refer to the specification for a description.
+This repository holds all interfaces related to [PSR-17 (HTTP Factories)][psr-url].
 
-You can find implementations of the specification by looking for packages providing the 
-[psr/http-factory-implementation](https://packagist.org/providers/psr/http-factory-implementation) virtual package.
+Note that this is not a HTTP Factory implementation of its own. It is merely interfaces that describe the components of a HTTP Factory.
 
-[psr-17]: https://www.php-fig.org/psr/psr-17/
+You can find [implementations][implementation-url] and [installation instructions][package-url] for the specification on the packagist.
```

Comment:
In here the reference to the virtual package is lost in the link

Reply:
```suggestion
You can find [implementations][implementation-url] of the specification on Packagist.
```